build.gradle

```diff
@@ -3,9 +3,11 @@ buildscript {
         corda_release_group = 'net.corda'
         corda_release_version = '4.3'
         tokens_release_group = 'com.r3.corda.lib.tokens'
-        tokens_release_version = '1.1-RC05'
+        tokens_release_version = '1.1'
```

Comment:
We should jump straight to the latest public version of the SDK. Right now I believe that's `1.2`

Reply:
Fixed

Reply:
What do I need to add to my `build.gradle` to resolve tokens 1.2? If I simply bump the version number I get this error:
```Could not resolve com.r3.corda.lib.tokens:tokens-selection:1.2.

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html

```

I've checked in [Artifactory](https://software.r3.com/artifactory/corda-lib-dev/com/r3/corda/lib/tokens/) and I can't seem to find the JARs

Reply:
Fixed